platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java

```diff
@@ -30,6 +29,11 @@ public static String getSystemIdentifier() {
 		return AGCoreModuleInstance.getInstance().getSystemName();
 	}
 
+	public Boolean isAdministrator(AGUser user) {
```

Comment:
If (!) we retain this method, we should rename it to `hasAdministrationPermission`. However, recommend inlining it.